### PR TITLE
Create simple disk usage command

### PR DESCRIPTION
**Pull request by WillGibson**

### CODE REVIEW

File: bin/docker-disk-usage [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env php
+<?php
+
+use BiffBangPow\DockerDiskUsage\DockerDiskUsage;
+
```
Comment (catharsisjelly) [inline bin/docker-disk-usage]: wondering if this might be helpful to you - http://docker-php.readthedocs.io/en/latest/basic/

Comment (WillGibson) [reply bin/docker-disk-usage]: This is good @catharsisjelly, took no time at all to get the volumes with that, thanks for the heads up. It seems we still need to execute a bash style command to get the six of the volumes, but getting the rest of the info is much cleaner. I'll crack on with the containers and images.

File: src/DockerDiskUsage.php [hunk 2]
```diff
@@ -0,0 +1,536 @@
...
+     */
+    private $useTextFormatting = self::FORMATTED_TEXT;
+
+    /**
+     * @return void
+     */
+    public function __construct()
+    {
+        $this->cliFormatter = new Color();
+        $this->docker = new Docker();
```
Comment (jamesbbp) [inline src/DockerDiskUsage.php:69]: Getting `Class 'Docker\Docker' not found in /home/james/Sites/docker-disk-usage/src/DockerDiskUsage.php:63`

Comment (WillGibson) [reply src/DockerDiskUsage.php:69]: Probably need to composer install again, I just started using that.

File: bin/docker-disk-usage (hunk 1 above)
Comment (WillGibson) [reply bin/docker-disk-usage]: @catharsisjelly I've gone as far as I can with Docker PHP for now if you want to have another look.

**Comment by WillGibson** [comment]
I've just noticed, now that I have several projects spun up locally, that since my work yesterday, all the containers are coming up under all the applications. I'll fix that in a bit.

Later... Done.

File: bin/docker-disk-usage [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+require_once __DIR__ . '/../vendor/autoload.php';
+
+$dockerDiskUsage = new DockerDiskUsage();
+
+$arguments = getopt('', ['help', 'detailed', 'plain-text', 'include-mounts']);
```
Comment (catharsisjelly) [inline bin/docker-disk-usage]: I also highly recommend - https://packagist.org/packages/ulrichsg/getopt-php

File: src/DockerDiskUsage.php [hunk 4]
```diff
@@ -0,0 +1,542 @@
...
+        $output .= $this->appendErrors();
+        return $output;
+    }
+
+    /**
+     * @return string
+     */
+    public function helpInformation(): string
+    {
+        return '
```
Comment (catharsisjelly) [inline src/DockerDiskUsage.php]: Can we use a HEREDOC? http://php.net/manual/en/language.types.string.php#language.types.string.syntax.heredoc

File: src/DockerDiskUsage.php [hunk 5]
```diff
@@ -0,0 +1,542 @@
...
+                foreach ($applicationData['mounts'] as $mountPath => $mountSize) {
+                    $mountsDiskUsage += $mountSize;
+                }
+                $totalApplicationDiskUsage += $mountsDiskUsage;
+            }
+
+            $output .= $this->bold($this->yellow(
+                $this->padApplicationName($applicationName) . $this->padDiskUsage($totalApplicationDiskUsage, '-')
+            ));
+            $output .= PHP_EOL . PHP_EOL;
```
Comment (catharsisjelly) [inline src/DockerDiskUsage.php]: Function out to `doubleLineReturn`?

Comment (WillGibson) [reply src/DockerDiskUsage.php]: Done something along these lines

File: src/DockerDiskUsage.php (hunk 4 above)
Comment (WillGibson) [reply src/DockerDiskUsage.php]: Not required now it's using GetOpt

File: bin/docker-disk-usage (hunk 3 above)
Comment (WillGibson) [reply bin/docker-disk-usage]: Thank you :-)

**Comment by catharsisjelly** [comment]
:+1:

File: bin/docker-disk-usage [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+
+require_once __DIR__ . '/../vendor/autoload.php';
+
+define('NAME', 'DockerDiskUsage');
+define('VERSION', '0.1.0');
+
+// Usage:
+// 	docker-disk-usage [OPTIONS]
+//
+// Show how much disk space your Docker "things" are using.
```
Comment (jamesbbp) [inline bin/docker-disk-usage]: Why is this stuff commented out?

Comment (WillGibson) [reply bin/docker-disk-usage]: I left it in for reference while I was changing things and forgot to delete it.

**Comment by WillGibson** [comment]
Thanks for your feedback @jamesbbp and @catharsisjelly. I'm going to squash this into a single commit now and release 0.1.0 :-)

**Pull request merged**